crypto/key.go

```diff
@@ -251,6 +251,18 @@ func (key *Key) GetPublicKey() (b []byte, err error) {
 
 // --- Key object properties
 
+// CanSign returns true if any of the Keys can be used for signing.
```

Comment:
Here either we change to "if any of the subkeys can" or "if the key can", since this method is used only on a single entity, and same for `CanEncrypt`

Reply:
Sure, I'll update it.